staging/src/k8s.io/kubectl/pkg/cmd/get/sorter.go

```diff
@@ -278,9 +278,6 @@ func isLess(i, j reflect.Value) (bool, error) {
 					return sortorder.NaturalLess(itype, jtype), nil
 				}
 				jtypeQuantity, _ := resource.ParseQuantity(jtype)
```

Comment:
Thanks for catching this! I wanted to catch the `err` but I forgot to add this. Can you change it to 
```suggestion
				jtypeQuantity, err := resource.ParseQuantity(jtype)
```

instead? Thanks

Reply:
Done.